### PR TITLE
Fix anchor select field component

### PR DESCRIPTION
**Pull request by GuySartorelli**
This should have been done when we upgraded react-select. My bad. The `Creatable` component is replaced with `CreatableSelect` and along with changes to that component's props, the selectors have also changed.

Fixes part of https://github.com/silverstripe/silverstripe-cms/actions/runs/3726652553/jobs/6320241058

> Element matching css "form#Form_editorAnchorLink" not found. (Behat\Mink\Exception\ElementNotFoundException)

## Note
Requires https://github.com/silverstripe/silverstripe-framework/pull/10635 as well, as the affected behat tests also include interacting with tree dropdowns.

## Parent issue
- https://github.com/silverstripeltd/product-issues/issues/655

### CODE REVIEW

File: client/src/components/AnchorSelectorField/AnchorSelectorField.js [hunk 1]
```diff
@@ -95,6 +95,7 @@ class AnchorSelectorField extends SilverStripeComponent {
    * @param {String} value
    */
   handleChange(value) {
+    console.log(value);
```
Comment (ChloeSartorelli) [inline client/src/components/AnchorSelectorField/AnchorSelectorField.js]:
```suggestion
```
Oops I need to remove this

**Comment by GuySartorelli** [comment]
Marking this as **DRAFT** as it requires https://github.com/silverstripe/webpack-config/pull/55 to be merged, tagged, and published in npm before I can add the emotion cache component.
Edit: Good to go now

**Comment by GuySartorelli** [comment]
The changes to `yarn.lock` are a result of running `yarn upgrade @silverstripe/webpack-config` so that I can use the `EmotionCssCacheProvider` exposed in the latest alpha release of that library.

**Comment by GuySartorelli** [comment]
Note that this PR now requires https://github.com/silverstripe/silverstripe-admin/pull/1432 to be merged before it will go "green" (behat will still fail, but the tests that this PR specifically is targetting should pass)

**Review by maxime-rainville — APPROVED** [review]
LGTM

**Pull request merged**